install/kots/manifests/kots-preflight.yaml

```diff
@@ -237,6 +261,30 @@ spec:
               message: The nodes must contain at least 4 cores
           - pass:
               message: There are at least 4 cores in the nodes
+    - nodeResources:
+        checkName: Memory per workload_meta node
+        filters:
+          selector:
+            matchLabel:
+              gitpod.io/workload_meta: "true"
+        outcomes:
+          - fail:
+              when: "min(memoryCapacity) < 16G"
```

Comment:
Isn't this a bit too high? 🤔 

From the https://github.com/gitpod-io/gitpod/issues/11344#issuecomment-1185493784, It does make sense but should we instead warn users on this? Fialing seems to be a bit hard requirement🤔

Reply:
@MrSimonEmms If think `< 4c/16GB` should be a warning. hard limit on `2c/8GB` make sense.